lib/rules/define-macros-order.js

```diff
@@ -142,7 +144,10 @@ function create(context) {
           program
         )
         const orderedList = order
-          .map((name) => ({ name, node: macrosNodes.get(name) }))
+          .flatMap((name) => {
+            const nodes = macrosNodes.get(name) ?? []
+            return nodes?.map((node) => ({ name, node }))
```

Comment:
This can't be undefined, since you are using `??` above.

```suggestion
            return nodes.map((node) => ({ name, node }))
```

Reply:
Good catch ;)